### ISSUE TITLE
V2.0.1.7.3 works but with huge lag and no cursor

### ISSUE THREAD

**Issue by ivanov04** [MindShow/USBDisplay#65]
Can you please freaking provide us with a new version or something that actually works?!
What is this? How are you selling something that doesn't work...

**Comment by SavostinVladimir** [comment]
Have the same issue!

**Comment by ivanov04** [comment]
Yeah I know, it's really frustrating, how can someone sell something that doesn't work and does NOTHING to fix that... Waste of money, they are not replying to issues and don't care anymore

**Comment by gwa100** [comment]
im using v1.0.0.18 with discrete gpu and its still laggy with big content on the screen, wtf is this shitty software
